### PR TITLE
Yarn dependencies

### PR DESCRIPTION
**Pull request by jdelbick**
## Description

Switching to yarn for installing dependencies since npm does not support resolutions ootb and the resolutions solution `npm-force-resolutions` fails on windows.

Fixes # https://jira.corp.adobe.com/browse/NUI-627

Yarn comes pre-installed on Linux in travis, but not in osx or windows so we have to install it manually. Also, `yarn install` hangs on windows because yarn is starts a GPG subprocess that never gets closed. The solution is to disable the gpg check for windows using the environment variable `YARN_GPG=no`: https://travis-ci.community/t/timeout-after-build-finished-and-succeeded/1336.

Looks like yarn had recognized the complaints from this issue: https://github.com/yarnpkg/website/issues/1029 but the PR in yarn never got merged so it seems the environment variable is the only solution

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -17,21 +17,27 @@ os:
...
+    - os: windows
+      node_js: "12"
+      env: TEST_DIR=server YARN_GPG=no
+    - os: windows
+      node_js: "14"
+      env: TEST_DIR=server YARN_GPG=no
+before_install:
+  - cd $TEST_DIR
+  - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.22.4
+  - export PATH="$HOME/.yarn/bin:$PATH"
```
Comment (tmathern) [inline .travis.yml:39]: Why is the font so strange?
Make sure to convert this to "simple" textfile before copy/pasting paths. Some tools on some operating systems would have issues to deal with this.

**Comment by jdelbick** [comment]
@tmathern btw, I switched both the server and client to use `yarn` for installing to make it easier.

File: .travis.yml (hunk 1 above)
Comment (jdelbick) [reply .travis.yml:39]: The font looks normal on my machine:
Screen Shot 2020-06-18 at 4 05 01 PM

not sure I totally understand what / how I should change this

**Comment by tmathern** [comment]
@jdelbick Aaaaah, right. I see the server/package-lock is deleted!

File: .travis.yml (hunk 1 above)
Comment (tmathern) [reply .travis.yml:39]: Funny. I see this on Github:
image

I hope it's just a rendering issue for their new design!

**Pull request merged**